packages/base-controller/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Changelog
```

Comment:
Now we're getting into the packages themselves. Due to the `paths` option we are setting for TypeScript and `moduleNameMapper` we are setting for Jest, I've chosen a standard nomenclature for the directories in `packages` — namely, that they should match the names of the packages on NPM, minus `@metamask/`. So:

* `packages/base-controller` -> `@metamask/base-controller`
* `packages/ens-controller` -> `@metamask/ens-controller`
* `packages/controller-utils` -> `@metamask/controller-utils`